views/drones/list.hbs

```diff
@@ -1 +1,15 @@
-<h2>Available drones</h2>
\ No newline at end of file
+<h2>Available drones</h2>
+
+{{#each drones}}
+
+    <div>
+        <h3>{{name}}</h3>
+        <p>Propellers: {{propellers}}</p>
+        <p>Max Speed: {{maxSpeed}}</p>
+         <a href="/drones/{{_id}}/edit">Edit Drone</a>
+          <form action="/drones/{{_id}}/delete" method="post">
+            <button type="submit">Delete Drone</button>
+        </form>
+    </div>
+
```

Comment:
Just one piece of advice, add buttons to all the pages when doing the lab for a better navigation experience! :)